src/main/java/meditracker/DailyMedicationManager.java

```diff
@@ -27,6 +27,18 @@ public DailyMedicationManager() {
         // TODO: Add @see Storage when implemented
     }
 
+    /**
+     * Constructs DailyMedicationManager with lines imported from the textfile
+     * @param lines lines of String read from each row in the textfile
```

Comment:
Javadoc Header comments there should be a blank line between the description and the parameter annotation (@) as per https://se-education.org/guides/conventions/java/basic.html#comments

Reply:
Thx, I'll add that line!